src/components/List.jsx

```diff
@@ -1,25 +1,34 @@
 import React from 'react';
+import { useNavigate } from 'react-router-dom';
 import db from '../lib/firebase';
 import { collection } from 'firebase/firestore';
 import { useCollection } from 'react-firebase-hooks/firestore';
 
 export const List = ({ token }) => {
+  let navigate = useNavigate();
```

Comment:
You should only use `let` when you are reassigning the variable somewhere else. In this case `const` would be more appropriate